docs/sdks/sdk-features/bandits.md

```diff
@@ -22,7 +22,11 @@ are strings, so bandits are used with string-typed flags. Actions and attributes
 When requesting an assignment from a flag with a bandit, the set of actions and their attributes are provided as an
 additional argument to `getStringAssignment()`.
 
-For example, in the Java SDK, the call may look like:
+:::info
+Depending on the SDK you are using, a `getBanditAction()` alternative method may be available. Refer to the [Node](/sdks/server-sdks/node/#usage-with-contextual-multi-armed-bandits) or [Python](https://docs.geteppo.com/sdks/server-sdks/python/#6-contextual-bandits) documentation for more details. 
```

Comment:
soon Java will have this too 📈 

I'll file a ticket to update these docs to conform to `getBanditAction()`